pkg/autodiscovery/listeners/kubelet.go

```diff
@@ -296,12 +297,48 @@ func (l *KubeletListener) createService(entity string, pod *kubelet.Pod, firstRu
 	}
 
 	l.m.Lock()
+	defer l.m.Unlock()
+	old, found := l.services[entity]
+	if found && kubeletSvcEqual(old, &svc) {
+		log.Debugf("Received a duplicated kubelet service '%s'", svc.entity)
```

Comment:
Is it a bug in this case? or it is possible in case of crashloop? 
If it is a case that can append, we should not print a `debug()` message. (only `Trace()`)